ESPixelStick/src/input/InputFPPRemotePlayListFsm.cpp

```diff
@@ -356,22 +356,21 @@ void fsm_PlayList_state_Paused::GetStatus (JsonObject& jsonStatus)
 {
     // DEBUG_START;
 
-    JsonObject PauseStatus = jsonStatus.createNestedObject (CN_Paused);
-    
+    JsonObject pauseStatus = jsonStatus.createNestedObject (CN_Paused);
```

Comment:
Please do not be changing these to lower case.

Reply:
Resolved in 7b4bf53